netplan_cli/cli/state.py

```diff
@@ -39,16 +40,19 @@
 DEVICE_TYPES = {
     'bond': 'bond',
     'bridge': 'bridge',
+    'dummy': 'dummy',
```

Comment:
nitpick: Should this be translated to `dummy-device` to stay in line with Netplan YAML?